src/components/AssetsDropZone/AssetsDropZone.scss

```diff
@@ -25,12 +26,16 @@
   }
 }
 
+.drop-zone-compact {
```

Comment:
Could these styles be generally applied when the screen in small? If so, media queries might be the way to go, e.g. https://github.com/edx/studio-frontend/blob/2dc352af824a6963a8b4ef905bbcfaec1407dc6e/src/components/EditImageModal/EditImageModal.scss#L69, with a mobile-first design.

Reply:
Perhaps? Do you mean to have it handle both for this instance + smaller screens? I'm not familiar enough to know if I use the media query and view the modal on my desktop if the UI will still leverage the compact styles? I'm assuming no, but happy to discuss this option more.

Reply:
I'm actually just thinking that you could combine the CSS with the above rather than having a separate selector. For the case of the modal, it would just end up being smaller. It's probably easier to show you what I mean. :)

Reply:
I just had an idea about this. You could have `drop-zone` inherit from `drop-zone-compact` and leave the media queries in `drop-zone`.